src/main/kotlin/no/nav/familie/ks/sak/api/dto/BisysDto.kt

```diff
@@ -5,20 +5,30 @@ import no.nav.familie.ks.sak.common.exception.Feil
 import org.springframework.http.HttpStatus
 import java.time.YearMonth
 
-data class BisysDto(val barnIdenter: List<String>) {
+data class BisysDto(val identer: List<String>) {
     init {
-        if (barnIdenter.any { it.length != 11 }) {
-            throw Feil("Ugyldig input. barnIdenter må være 11 siffer", httpStatus = HttpStatus.BAD_REQUEST)
+        if (identer.any { it.length != 11 }) {
+            throw Feil("Ugyldig input. identerdenter må være 11 siffer", httpStatus = HttpStatus.BAD_REQUEST)
         }
     }
 }
 
-data class BisysResponsDto(val utbetalingsinfo: Map<String, List<UtbetalingsinfoDto>>)
+data class BisysResponsDto(val infotrygdPeriode: List<InfotrygdPeriode>?, val ksSakPeriode: List<KsSakPeriode>?)
```

Comment:
```suggestion
data class BisysResponsDto(val infotrygdPeriode: List<InfotrygdPeriode>?, val ksSakPerioder: List<KsSakPeriode>?)
```
Kanskje rename til perioder gitt at det er flertall (liste), eller er navnet fastbestemt av bisys? 😛

Reply:
Enig, Men tenker egentlig det er fint med enda mer beskrivende navn feks. `PerioderMedKontantsøtteForBisys`